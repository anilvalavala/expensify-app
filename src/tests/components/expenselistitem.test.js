import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';

test('Test expense list item', () => {
    const expense = {
        id: "1",
        description: "Testing from Enzyme",
        amount: '100',
        createdAt: '0'
    };
    const wrapper = shallow(<ExpenseListItem {...expense} />)
    expect(wrapper).toMatchSnapshot();
});