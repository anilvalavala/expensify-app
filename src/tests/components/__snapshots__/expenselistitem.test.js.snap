// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test expense list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Testing from Enzyme
    </h3>
  </Link>
  <p>
    100
     - 
    0
  </p>
</div>
`;
